content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Sail operator 1.0.0 has been released: manage Istio with an operator"
+description: Sail Operator 1.0.0 released, a project part of the istio-ecosystem organization. Let's dive in into the basics of the Sail Operator and let's run an example to show how easy it is to manage Istio with it.
+publishdate: 2025-03-19
+attribution: "Francisco Herrera - Red Hat"
+keywords: [istio,operator,sail,incluster,istiooperator]
+---
+
+The Sail operator is GA with a clear mission: to simplify and streamline Istio management in your cluster. By using the Kubernetes operator pattern, Sail operator aims to add new value to istio by improving the upgrade process.
+
+## Simplified Deployment & Management
+
+The Sail operator is engineered to cut down the complexity of installing and running Istio. It automates manual tasks, ensuring a consistent, reliable, and uncomplicated experience from initial installation to ongoing maintenance and upgrades of Istio versions in your cluster. Besides this, the Sail operator APIs are built around Istio’s Helm chart APIs, which means that all the istio configurations are available through the Sail Operator CRDs’ values.
+
+We encourage users to go through our live [documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs) to learn more about this new way to manage your istio control plane.
+
+The main resources that are part of the Sail operator are:
+* `Istio`: manages your istio control plane.
+* `IstioRevision`: it represents a revision of the control plane.
+* `IstioRevisionTag`: resource represents a stable revision tag, which functions as an alias for Istio control plane revisions.
+* `IstioCNI`: Istio's CNI plugin resource.
+* `ZTunnel`: ambient mode ztunnel `DaemonSet` (alpha feature).
+
+Note that if you are migrating from the (now deprecated and deleted) In-Cluster Istio operator, you can check this section in our [documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#migrating-from-istio-in-cluster-operator) where we explain the equivalence of resources, or you can also try our [resource converter](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#converter-script) to easily convert your `IstioOperator` resource to an `Istio` resource.
```

Comment:
```suggestion
{{< idea >}}
If you are migrating from the [since-removed Istio in-cluster operator](https://istio.io/latest/blog/2024/in-cluster-operator-deprecation-announcement/), you can check this section in our [documentation](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#migrating-from-istio-in-cluster-operator) where we explain the equivalence of resources, or you can also try our [resource converter](https://github.com/istio-ecosystem/sail-operator/tree/main/docs#converter-script) to easily convert your `IstioOperator` resource to an `Istio` resource.
{{< /idea >}}
```